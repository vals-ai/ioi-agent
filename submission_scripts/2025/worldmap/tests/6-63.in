version https://git-lfs.github.com/spec/v1
oid sha256:c19fd0a1456b0bc184f4828b27e0a7501da4dfe110fdb16a06b06ecd0fd506b6
size 2448
