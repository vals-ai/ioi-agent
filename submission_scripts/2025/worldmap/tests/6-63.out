version https://git-lfs.github.com/spec/v1
oid sha256:09f8a980f83b49f687304321dfc1258ec6191abb29dfba1ef943c516c26e8e51
size 189650
